### PR TITLE
Adding a delay of 1 sec in retry loops

### PR DESCRIPTION
**Pull request by dimpar**
Closes: https://github.com/keep-network/keep-ecdsa/issues/395

Adding a short delay interval of 1 sec. upon failure when calculating a signature and signer generation.

### CODE REVIEW

File: pkg/node/node.go [hunk 1]
```diff
@@ -255,6 +257,7 @@ func (n *Node) CalculateSignature(
 				keepAddress.String(),
 				err,
 			)
+			time.Sleep(time.Second)
```
Comment (nkuba) [inline pkg/node/node.go:260]: Maybe we could extract the value to a constant defined on top of the file and reuse it?

Comment (dimpar) [reply pkg/node/node.go:260]: Hm, I guess I'm open, but what do you think we should name this constant? We use `time.Second` in 3 different places in slightly different contexts and it is already a constant. Naming `const second = time.Second` feels redundant. Smth general like `contextWaitInterval`?
Also we have similar case in `publishSignature` -> `time.Sleep(1 * time.Minute)`, but this one is used only in 1 place.

Comment (nkuba) [reply pkg/node/node.go:260]: All the three places are related, maybe we could name it just `retryDelay`?

The value in `publishSignature` is a different story and can remain as is.

Comment (dimpar) [reply pkg/node/node.go]: done

**Comment by nkuba** [comment]
In the other PR we're adding yet another step with `continue` we will need to cover with a changes from this PR: https://github.com/keep-network/keep-ecdsa/pull/412/files#diff-440b0b0f55b1efa66d8a9a49cf4df99eR311

**Comment by nkuba** [comment]
We should consider adding the delay in every retry loop we do `continue`. An elegant solution would be having exponential backoff proposed by @Shadowfiend (e.g. https://github.com/cenkalti/backoff).

For this PR we could start with a simple 1-second sleeps and TODOs left in the code. Then it could be replaced with the backoff.

**Comment by nkuba** [comment]
@pdyraga @dimpar I introduced some updates to this PR. I would love to have you guys reviewed this PR.

**Pull request merged**